pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -94,6 +94,7 @@ public abstract class BaseBrokerRequestHandler implements BrokerRequestHandler {
   private static final Logger LOGGER = LoggerFactory.getLogger(BaseBrokerRequestHandler.class);
   private static final String IN_SUBQUERY = "insubquery";
   private static final String IN_ID_SET = "inidset";
+  private static final boolean DEFAULT_DISABLE_GROOVY = true;
```

Comment:
Let's put this constant along with the `DISABLE_GROOVY` in `CommonConstants.Broker`

Reply:
moved